packages/documentation-v7/src/stories/components/switch/switch.stories.ts

```diff
@@ -0,0 +1,237 @@
+        label: {
+          both: 'Both',
+          before: 'Before',
+          after: 'After',
+        },
+      },
+      options: ['both', 'before', 'after'],
+      table: {
+        category: 'General',
+      },
+    },
+    labelBefore: {
+      name: 'Label (before)',
+      description: 'Describes the content/topic of the component.',
+      if: {
+        arg: 'labelPosition',
+        neq: 'after',
+      },
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'General',
+      },
+    },
+    labelAfter: {
+      name: 'Label (after)',
+      description: 'Describes the content/topic of the component.',
+      if: {
+        arg: 'labelPosition',
+        neq: 'before',
+      },
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'General',
+      },
+    },
+    hiddenLabel: {
+      name: 'Hidden Label',
+      description:
+        'Renders the component with or without a visible label.<span className="mt-mini alert alert-info alert-sm">There are accessibility concerns with hidden labels.<br/>Please read our <a href="/?path=/story/foundations-accessibility--page#labels">label accessibility guide</a>.</span>',
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        category: 'General',
+      },
+    },
+    checked: {
+      name: 'Checked',
+      description: 'When set to `true`, places the component in the checked state.',
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        category: 'States',
+      },
+    },
+    disabled: {
+      name: 'Disabled',
+      description:
+        'When set to `true`, disables the component\'s functionality and places it in a disabled state.<span className="mt-mini alert alert-info alert-sm">There are accessibility concerns with the disabled state.<br/>Please read our <a href="/?path=/docs/foundations-accessibility--page#disabled-state">disabled state accessibility guide</a>.</span>',
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        category: 'States',
+      },
+    },
+    validation: {
+      name: 'Validation',
+      description: "Controls the display of the component's validation state.",
+      control: {
+        type: 'radio',
+        labels: {
+          'null': 'Default',
+          'is-valid': 'Valid',
+          'is-invalid': 'Invalid',
+        },
+      },
+      options: ['null', 'is-valid', 'is-invalid'],
+      table: {
+        category: 'States',
+      },
+    },
+  },
+};
+
+export default meta;
+
+const VALIDATION_STATE_MAP: Record<string, undefined | boolean> = {
+  'null': undefined,
+  'is-valid': false,
+  'is-invalid': true,
+};
+
+function getValidationFeedback({ validation }: Args) {
+  return validation !== 'null'
+    ? html`
+        <p class=${validation.split('-')[1] + '-feedback'}>
+          ${validation === 'is-valid' ? 'Ggranda sukceso!' : 'Eraro okazis!'}
+        </p>
+      `
+    : null;
+}
```

Comment:
You do not need to call a function for this.
Include this `const` into the render function `renderSwitch`.